.github/maintainers_guide.md

```diff
@@ -4,15 +4,15 @@ This document describes tools, tasks and workflow that one needs to be familiar
 this project. If you use this package within your own software as is but don't plan on modifying it, this guide is
 **not** for you.
 
-## Tools (optional)
+## Tools
 
 All you need to work on this project is a recent version of [Deno](https://deno.land/)
 
 ## Tasks
 
 ### Building
 
-* The majority of this library (`src/generated`) is generated code based off of a slack api spec file. Updating the web apis based off a new spec file can be done by getting the latest spec file from https://api.slack.com/web#spec or via the [slack-api-specs](https://github.com/slackapi/slack-api-specs) repo directly. 
+* The majority of this library (`src/generated`) is generated code based off of a slack api spec file. Updating the web apis based off a new spec file can be done by getting the latest spec file from https://api.slack.com/web#spec or via the [slack-api-specs](https://github.com/slackapi/slack-api-specs) repo directly.
```

Comment:
Seeing this white-space change made me realize this is no longer true. This code is generated off of [a file](https://github.com/slackapi/deno-slack-api/blob/main/scripts/src/public-api-methods.ts) that has a list of api methods. I'm happy to update that detail in a followup PR though, as it's been wrong here, and you were just fixing some white-space here 😄

Reply:
I added a bit of detail in this PR, but if it is incomplete or insufficient, we can follow up with another!